hack/xref-helpmsgs-manpages

```diff
@@ -23,17 +24,17 @@ $| = 1;
 # BEGIN user-customizable section
 
 # Path to podman executable
-my $Default_Podman = './bin/podman';
+my $Default_Podman = "$FindBin::Bin/../bin/podman";
 my $PODMAN = $ENV{PODMAN} || $Default_Podman;
 
 # Path to all doc files, including .rst and (down one level) markdown
-my $Docs_Path = 'docs/source';
+our $Docs_Path = 'docs/source';
```

Comment:
`my` -> `our` makes it a scoped named variable that we can then use from the `.t` file.